easybuild/easyconfigs/a/Arrow/Arrow-0.12.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = 'CMakeMake'
+
+name = 'Arrow'
+version = '0.12.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://arrow.apache.org'
+description = "Apache Arrow (incl. PyArrow Python bindings)), a cross-language development platform for in-memory data."
```

Comment:
@boegel nitpicking but could you use triple quotes here for consistency?

Reply:
I usually use `"..."` when the description fits on a single line (the one reason to use `"""` is if it doesn't fit on a single line)